README.rst

```diff
@@ -93,8 +93,8 @@ Then add this line to your ``.zshrc``
 Source the plugin shell script in your `~/.zshrc` profile. For example
 
 ::
-
-   source $HOME/zsh-autoswitch-virtualenv/autoswitch_virtualenv.plugin.zsh
+   git clone "https://github.com/MichaelAquilina/zsh-autoswitch-virtualenv.git" "$ZSH_CUSTOM/plugins/autoswitch_virtualenv"
```

Comment:
Maybe we can make it a two line step. Something roughly like

1. Clone zsh-autoswitch-vritualenv if you havent already
2. Add the line below to your zshrc file

Reply:
How about saying
```
Clone the repository as per for oh-my-zsh and then add the following line to your .zshrc file:

source <git-clone-directory>/autoswitch_virtualenv.plugin.zsh
```